nimterop/cimport.nim

```diff
@@ -102,6 +102,9 @@ proc getToast(fullpath: string, recurse: bool = false): string =
   if gStateCT.symOverride.len != 0:
     cmd.add &"--symOverride={gStateCT.symOverride.join(\",\")} "
 
+  if gStateCT.pluginFile.nBl and gStateCT.pluginFile.fileExists():
```

Comment:
safety first!
```
if gStateCT.pluginFile.nBl:
  doAssert gStateCT.pluginFile.fileExists()
  ...
```

Reply:
i don't follow; if `gStateCT.pluginFile` is nonempty but invalid, u're currently just silently skipping it and there'll be crash down the line.

Reply:
I got it - moved it into `cPlugin()` itself.

Reply:
[RESOLVED]